.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
please also add .vs folder to the gitignore. don't want editor config or binary files checked in.